package.json

```diff
@@ -152,6 +152,7 @@
     "prettier": "1.19.1",
     "pretty-bytes": "5.3.0",
     "pumpify": "2.0.1",
+    "puppeteer": "2.0.0",
```

Comment:
We don't want to include `puppeteer` as a dependency in the root `package.json` because it adds a few hundred MBs, which most AMP developers won't use locally. See how we install it as a task-specific dependency with visual-diff or e2e tests, which have their own `package.json` files